lib/jekyll/gitlab/letsencrypt/configuration.rb

```diff
@@ -40,7 +40,7 @@ def layout
           end
 
           def personal_access_token
-            jekyll_config['personal_access_token']
+            ENV['GITLAB_TOKEN'] || jekyll_config['personal_access_token']
```

Comment:
I would test for emptiness, this now works only for unset variables. This is still great but this is the rare case when:

```
[alkuzad@localhost ~]$ GITLAB_TOKEN="LOL" ruby -e "p ENV['GITLAB_TOKEN'] || 'test'"
"LOL"
[alkuzad@localhost ~]$ GITLAB_TOKEN="" ruby -e "p ENV['GITLAB_TOKEN'] || 'test'"
""
[alkuzad@localhost ~]$ GITLAB_TOKEN= ruby -e "p ENV['GITLAB_TOKEN'] || 'test'"
""
[alkuzad@localhost ~]$ ruby -e "p ENV['GITLAB_TOKEN'] || 'test'"
"test"
```

Yeah ... I love bash too ...